tools/analysis/eval_metric.py

```diff
@@ -0,0 +1,70 @@
+import argparse
+
+import mmcv
+from mmcv import Config, DictAction
+
+from mmaction.datasets import build_dataset
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description='Evaluate metric of the '
+                                     'results saved in pkl/yaml/json format')
+    parser.add_argument('config', help='Config of the model')
+    parser.add_argument('results', help='Results in pkl/yaml/json format')
+    parser.add_argument(
+        '--eval',
+        type=str,
+        nargs='+',
+        help='evaluation metrics, which depends on the dataset, e.g.,'
+        ' "top_k_accuracy", "mean_class_accuracy" for video dataset')
+    parser.add_argument(
+        '--cfg-options',
+        nargs='+',
+        action=DictAction,
+        default={},
+        help='override some settings in the used config, the key-value pair '
+        'in xxx=yyy format will be merged into config file. For example, '
+        "'--cfg-options model.backbone.depth=18 model.backbone.with_cp=True'")
+    parser.add_argument(
+        '--eval-options',
+        nargs='+',
+        action=DictAction,
+        help='custom options for evaluation, the key-value pair in xxx=yyy '
+        'format will be kwargs for dataset.evaluate() function')
+    args = parser.parse_args()
+    return args
+
+
+def main():
+    args = parse_args()
+
+    cfg = Config.fromfile(args.config)
+
+    # import modules from string list.
```

Comment:
so can this be replaced by the new `fromfile`?

Reply:
Done